src/model/claims.rs

```diff
@@ -1,15 +1,20 @@
-use serde::{ser::SerializeMap, Deserialize, Serialize, Serializer};
 use std::fmt::{Display, Formatter};
 
+use serde::{ser::SerializeMap, Deserialize, Serialize, Serializer};
+use uuid::Uuid;
+
 use crate::config::{CustomField, CustomFieldValue};
 
 #[derive(Debug, Deserialize)]
 pub struct Claims {
+    iss: String,
+    sub: String,
     aud: String,
-    iat: Option<i64>,
     exp: Option<i64>,
+    nbf: Option<i64>,
+    iat: Option<i64>,
+    jti: String,
```

Comment:
Are these fields exposed...? :thinking:

Reply:
They are not, but it's not necessary since they are serialized into a json and returned to the caller